spring-integration-amqp/src/main/resources/org/springframework/integration/amqp/config/spring-integration-amqp-4.2.xsd

```diff
@@ -229,6 +229,29 @@
 							]]></xsd:documentation>
 						</xsd:annotation>
 					</xsd:attribute>
+					<xsd:attribute name="amqp-template" type="xsd:string">
+						<xsd:annotation>
+							<xsd:appinfo>
+								<tool:annotation kind="ref">
+									<tool:expected-type type="org.springframework.amqp.core.AmqpTemplate" />
+								</tool:annotation>
+							</xsd:appinfo>
+							<xsd:documentation>
+								The AmqpTemplate bean reference to be used for sending replies.
+								Defaults to `RabbitTemplate` based on the provided `ConnectionFactory`.
+							</xsd:documentation>
+						</xsd:annotation>
+					</xsd:attribute>
+					<xsd:attribute name="default-reply-to">
+						<xsd:annotation>
+							<xsd:documentation>
+								The 'defaultReplyTo' address in the form '(exchange)/(routingKey)'
```

Comment:
(or `(queueName)` - in which case the default exchange will be used with the queue name as the routing key.